examples/CivisML_parallel_training.ipynb

```diff
@@ -153,7 +153,7 @@
     "\n",
     "print(\"Shape: \", df.shape)\n",
     "\n",
-    "df['is_cancer'] = [1 if val == 4 else 0 for val in df['is_cancer'] ]\n",
+    "df['is_cancer'] = [1 if val == 4 else (0 if val == 2 else '?') for val in df['is_cancer'] ]\n",
```

Comment:
Thanks for the PR! Perhaps `np.nan` instead of `'?'`? That's a more usual way to represent nulls in modeling.